packages/functional-tests/tests/subscription-tests/coupon.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { test, expect } from '../../lib/fixtures/standard';
+
+test.describe('severity-1', () => {
+  test('subscribe with an expired coupon', async ({
+    pages: { relier, login, subscribe },
+  }, { project }) => {
+    test.skip(project.name === 'production', 'prod needs a valid credit card');
+    test.skip(project.name === 'local', 'No need to be run on local');
+    test.slow();
+    await relier.goto();
+    await relier.clickSubscribe6month();
+    await subscribe.setFullName();
+    await subscribe.setCreditCardInfo();
+    await subscribe.addExpiredCoupon();
+
+    // Verifying the correct error message
+    expect(await subscribe.couponErrorMessageText()).toMatch(
+      'The code you entered has expired'
+    );
+  });
+
+  test.describe('severity-1', () => {
+    test('subscribe with an invalid coupon', async ({
+      pages: { relier, login, subscribe },
+    }, { project }) => {
+      test.skip(
+        project.name === 'production',
+        'prod needs a valid credit card'
+      );
+      test.skip(project.name === 'local', 'No need to be run on local');
+      test.slow();
+      await relier.goto();
+      await relier.clickSubscribe6month();
```

Comment:
nit: should it be `.clickSubscribe6Month()` (uppercase "M") or... rename to `.clickSubscribeSixMonth()`?
I'd have to ~Google~ Bing to see if there are any rules for PascalCase or camelCase rules when using digits in a variable name.